packages/edit-site/src/components/site-hub/style.scss

```diff
@@ -19,6 +19,30 @@
 .edit-site-site-hub__text-content {
 	// Necessary for the ellipsis to work.
 	overflow: hidden;
+	.edit-site-site-hub__site-view-link {
+		flex-grow: 0;
+		@include break-mobile() {
+			opacity: 0;
+			transition: opacity 0.2s ease-in-out;
+		}
+		&:focus {
+			outline: none;
+			box-shadow: none;
+		}
+		svg {
+			fill: $white;
+		}
+	}
+	&:hover {
+		.edit-site-site-hub__site-view-link {
+			opacity: 1;
+		}
+	}
+	&:focus {
+		.edit-site-site-hub__site-view-link {
+			opacity: 1;
+		}
+	}
```

Comment:
> I had trouble getting that to focus using the keyboard

The focus style is on the wrong element:

```diff
diff --git a/packages/edit-site/src/components/site-hub/style.scss b/packages/edit-site/src/components/site-hub/style.scss
index d38ca4068b..e8997fb37c 100644
--- a/packages/edit-site/src/components/site-hub/style.scss
+++ b/packages/edit-site/src/components/site-hub/style.scss
@@ -28,6 +28,7 @@
 		&:focus {
 			outline: none;
 			box-shadow: none;
+			opacity: 1;
 		}
 		svg {
 			fill: $white;
@@ -38,11 +39,6 @@
 			opacity: 1;
 		}
 	}
-	&:focus {
-		.edit-site-site-hub__site-view-link {
-			opacity: 1;
-		}
-	}
 }
 
 .edit-site-site-hub__title {
```

Reply:
Thanks, fixed.